### PR TITLE
Use _eq if field name is id, as ransack doesn't support id_cont

### PR DESCRIPTION
**Pull request by mabdelfattah**
https://github.com/activerecord-hackery/ransack/issues/85

### CODE REVIEW

File: app/assets/javascripts/activeadmin_addons/select2.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ $(function() {
         data: function(term) {
           var textQuery = { m: 'or' };
           fields.forEach(function(field) {
-            textQuery[field + '_contains'] = term;
+            field == "id" ? textQuery[field + '_eq'] = term : textQuery[field + '_contains'] = term;
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/select2.js]: Expected an assignment or function call and instead saw an expression.

File: app/assets/javascripts/activeadmin_addons/select2.js [hunk 2]
```diff
@@ -91,7 +91,11 @@ $(function() {
         data: function(term) {
           var textQuery = { m: 'or' };
           fields.forEach(function(field) {
-            textQuery[field + '_contains'] = term;
+            if (field == "id") {
+              textQuery[field + '_eq'] = term
+            } else {
+              textQuery[field + '_contains'] = term
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/select2.js]: Missing semicolon.

File: app/assets/javascripts/activeadmin_addons/select2.js [hunk 3]
```diff
@@ -91,7 +91,11 @@ $(function() {
         data: function(term) {
           var textQuery = { m: 'or' };
           fields.forEach(function(field) {
-            textQuery[field + '_contains'] = term;
+            if (field == "id") {
+              textQuery[field + '_eq'] = term
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/select2.js]: Missing semicolon.

**Comment by emilioeduardob** [comment]
I think we should merge this, what do you think @ldlsegovia ?

**Pull request merged**